scss/components/_downloads.scss

```diff
@@ -0,0 +1,53 @@
+// Downloads
+
+.downloads {
+  background-color: #efdecd; // Rename to variable
```

Comment:
Color literals like `#efdecd` should only be used in variable declarations; they should be referred to via variable everywhere else.